patches/@openeth+truffle-typings+0.0.6.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
```

Comment:
Is there an issue to track if this is gonna be patched? In case its been patched on the dependency level but we continue to patch it...

Reply:
Erm no issue to track, I tried asking developers on opengsn on group chat for help but no response. The issue seems to be global typing of mocha and chai imported from opengsn using `@openeth/truffle-typings` is messing with the type of globally inject jest causing lint to fail (test will be passing). Should I open an issue for it on opengsn?

Reply:
Yes please

Reply:
okay done https://github.com/opengsn/gsn/issues/537

Reply:
hmm where can I put this to track? should i open a issue on our repo also?

Reply:
Yes pls

Reply:
For linking: https://github.com/microsoft/TypeScript/issues/18588#issuecomment-652019495

Reply:
For linking/tracking: https://github.com/TradeTrust/document-creator-website/issues/86